src/Command/DoctrineEncryptDatabaseCommand.php

```diff
@@ -2,19 +2,21 @@
 
 namespace Ambta\DoctrineEncryptBundle\Command;
 
+use Ambta\DoctrineEncryptBundle\AmbtaDoctrineEncryptBundle;
 use Ambta\DoctrineEncryptBundle\DependencyInjection\DoctrineEncryptExtension;
+use Doctrine\DBAL\Platforms\MySQL80Platform;
+use Doctrine\DBAL\Types\Type;
+use Doctrine\ORM\Mapping\ClassMetadataInfo;
 use Symfony\Component\Console\Helper\ProgressBar;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\PropertyAccess\PropertyAccess;
 
 /**
  * Batch encryption for the database.
- *
- * @author Marcel van Nuil <marcel@ambta.com>
- * @author Michael Feinbier <michael@feinbier.net>
  */
 class DoctrineEncryptDatabaseCommand extends AbstractCommand
 {
```

Comment:
I feel like both the encrypt- and decrypt-command are a bit heavy on business logic and should be refactored in sunch a way that they call a service, which does the heavy lifting and have the service be properly unittested.

That said, I think that should be an issue on it's own and should not be fixed in this PR.